server/lib/NumberToText.js

```diff
@@ -22,12 +22,12 @@ exports.convert = convert;
 *  More about those after we take a look at numToWords itself …
 */
 
-function convert(input, lang, currencyName) {
+function convert(input, lang, currencyName, _dictionary) {
```

Comment:
Instead of modifying this function, let's just overwrite the `util.loadDictionary()` function.  Like this:

```
const numberToText = require('../../server/lib/NumberToText'); 
const util = require('../../server/lib/util');

util.loadDictionary = lang => lang === 'fr' ? fr : en;
```

I think that should be enough work to make this test work without modifying the underlying code.